rock_paper_scissors/api/models.py

```diff
@@ -0,0 +1,56 @@
+from sqlalchemy import Column, Integer, String
```

Comment:
I was wondering about that. I didn't know if imports had different structural purposes in a program if they needed to be imported differently...typing this out made me realize that this is not the case, as the interpreter wouldn't need that specification :smiley: